src/game/default/g_weapon.c

```diff
@@ -516,11 +542,11 @@ void G_FireBlaster(g_entity_t *ent) {
 
 		G_InitProjectile(ent, forward, right, up, org, 1.0);
 
-		G_BlasterProjectile(ent, org, forward, 1000, 15, 2);
+		G_BlasterProjectile(ent, org, forward, 1200, 15, 2);
```

Comment:
Speeding up the blaster, huh? I like it.

Reply:
Actually that wasn't intentional I think :P After this is merged I'll rebalance a couple weapons, namely the shotguns, MG, LG, and blaster. HB could do with a couple minor changes too.